src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -64,6 +68,38 @@ public String[] LiteralPath
         public Stream InputStream { get; set; }
 
         /// <summary>
+        /// InputString parameter
+        /// The strings to calculate a hash
+        /// We allow `null` and `empty` strings because we can get it from pipeline.
+        /// </summary>
+        /// <value></value>
+        [Parameter(Mandatory = true, ParameterSetName = StringHashParameterSet, Position = 0, ValueFromPipeline = true)]
+        [AllowNull()]
+        [AllowEmptyString()]
+        public string[] InputString { get; set; }
+
+        /// <summary>
+        /// Encoding parameter
+        /// The Encoding of the 'InputString'
```

Comment:
I'll fix the comment.

About #4119. We have a conflict. We can have (1) split into two cmdlets - `Get-FileHash` and `Get-StringHash`, two parameters - `FileEncoding` and `StringEncoding`, or universal cmdlet `Get-Hash` with a special solution. If we want one cmdlet it would be good that it accept strings "as is".